remedy.lic

```diff
@@ -12,6 +12,7 @@ class Remedy
   include DRCC
   include DRCI
   include DRCA
+  include DRCT
```

Comment:
❌ You'll need to add `common-travel` on line 8

Reply:
is there a way to add that while it's in a pull request?

Reply:
edit it in your branch aka https://github.com/dakre18/dr-scripts/tree/remedy-updates and it’ll update here

Reply:
Done, let me know if i need to do anything else.